release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4185,19 +4185,19 @@ $ oc adm release info 4.13.43 --pullspecs
+* Previously, if you need the *Form* view in the {product-title} web console to remove an alternate service from a `Route` resource, the alternate service remained in the cluster. With this release, if you delete an alternate service in this way, the alternate service is fully removed from the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-33650[OCPBUGS-33650])
 
-* Previously, if you configured an {product-title} cluster with a high number of internal services or user-managed load balancer IP addresses, you experienced a delayed startup time for the OVN-Kubernetes service. This delay occurred when the OVN-Kubernetes service attempted to install `iptables` rules on a node. With this release, the OVN-Kubernetes service can process a large number of services in a few seconds. Additionally, you can access a new log to view the status of installing `iptables` rules on a node. (link:https://issues.redhat.com/browse/OCPBUGS-33730[*OCPBUGS-33730*])
+* Previously, if you configured an {product-title} cluster with a high number of internal services or user-managed load balancer IP addresses, you experienced a delayed startup time for the OVN-Kubernetes service. This delay occurred when the OVN-Kubernetes service attempted to install `iptables` rules on a node. With this release, the OVN-Kubernetes service can process a large number of services in a few seconds. Additionally, you can access a new log to view the status of installing `iptables` rules on a node. (link:https://issues.redhat.com/browse/OCPBUGS-33730[OCPBUGS-33730])
 
-* Previously, the installation program did not remove the tag category from vSphere when it deleted a cluster. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33732[*OCPBUGS-33732*])
+* Previously, the installation program did not remove the tag category from vSphere when it deleted a cluster. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33732[OCPBUGS-33732])
 
-* Previously, the *Topology* view in the {product-title} web console did not show the visual connector between a virtual machine (VM) node and other non-VM components. With this release, the visual connector shows interaction activity of a component.  (link:https://issues.redhat.com/browse/OCPBUGS-33749[*OCPBUGS-33749*])
+* Previously, the *Topology* view in the {product-title} web console did not show the visual connector between a virtual machine (VM) node and other non-VM components. With this release, the visual connector shows interaction activity of a component.  (link:https://issues.redhat.com/browse/OCPBUGS-33749[OCPBUGS-33749])
```

Comment:
```suggestion
* Previously, the *Topology* view in the {product-title} web console did not show the visual connector between a virtual machine (VM) node and other non-VM components. With this release, the visual connector shows interaction activity of a component. (link:https://issues.redhat.com/browse/OCPBUGS-33749[OCPBUGS-33749])
```
I know it's not part of the changes introduced here, but removing the extra space is an easy fix.

Reply:
Well spotted. Removed the extra whitespace.